pkgs/development/python-modules/jax/default.nix

```diff
@@ -29,31 +30,21 @@ buildPythonPackage rec {
     owner = "google";
     repo = pname;
     rev = "jax-v${version}";
-    hash = "sha256-eGdAEZFHadNTHgciP4KMYHdwksz9g6un0Ar+A/KV5TE=";
+    hash = "sha256-pIl7zzl82w5HHnJadH2vtCT4mYFd5YmM9iHC2GoJD6s=";
   };
 
-  patches = [
-    # See https://github.com/google/jax/issues/7944
-    ./cache-fix.patch
```

Comment:
The linked issue is not closed, but the patch no longer applies and the caching code seems to have changed significantly, so I'm leaning toward thinking this is no longer an issue.

Reply:
Update: linked issue is closed now